raster-foundry/raster-foundry
# Add map token ogc auth

## Overview

This PR makes it so that wms and wcs services rewrite requests to `/<service>/<projectId>/map-token/<mapTokenId>` to `/<service>/<projectId>?mapToken=<mapTokenId>`, allowing wcs and wms service access for private projects.

### Checklist


### Notes

It wound up being kind of a pain to get autoslash and the request rewrite to play nicely together. I'm not sure why. But anyway, it's time to move on to other OGC features, like making maps show up.

## Testing Instructions

 * share a project with a map token
 * add that project as a wms and wcs layer in qgis, with `/map-token/<mapTokenId>` in the service route
 * you should be able to see the layers

Closes #4775
